utils/strings/validations.ts

```diff
@@ -1,3 +1,14 @@
 export function isEmpty(value: string) {
   return !value.trim();
 }
+
+// Note: This denotes a url as valid even without a protocol or domain.
+export function isValidUrl(value: string) {
+  const patternWithProtocol = new RegExp(
+    '^(http[s]?:\\/\\/(www\\.)?|ftp:\\/\\/(www\\.)?|www\\.){1}([0-9A-Za-z-\\.@:%_+~#=]+)+((\\.[a-zA-Z]{2,3})+)(/(.)*)?(\\?(.)*)?',
```

Comment:
Are we deliberately supporting subdirectories? e.g. `google.com/dslkfjasdlkfjsdkf/sdfkdjsf` will pass.

Reply:
I fixed it to disallow subdirectories.